assessments/CLI_CIRCLE/templates/studentAssessments.jsont

```diff
@@ -0,0 +1,127 @@
+[
+    "studentAssessmentIdentifier": "{{ student_wave_id }}__{{ subject_name }}",
+    "assessmentReference": {
+      "assessmentIdentifier": "cli_circle",
+      "namespace": "uri://www.cli.org/circle/Assessment"
+    },
+    "schoolYearTypeReference": {
+      "schoolYear": {{ school_year }}
+    },
+    "studentReference": {
+      "studentUniqueId": "{{ local_student_id }}"
```

Comment:
we want student_state_id to align with Ed-Fi

Reply:
Got it, so does Ed-Fi use `student_state_id` to identify students?

Reply:
yes, it's standard from the state to use the TSDS ID, which is the official state student id, and should match up with state ID here (I think we did a check on this awhile back)